config/src/test/java/org/hyperledger/besu/config/JsonGenesisConfigOptionsTest.java

```diff
@@ -116,4 +116,40 @@ public void configWithAnIbftWithNoValidatorsListedIsValid() {
     assertThat(configOptions.getTransitions().getIbftForks().get(1).getValidators().get().size())
         .isEqualTo(1);
   }
+
+  @Test
+  public void configWithValidIbftBlockRewardIsParsable() {
```

Comment:
Can you also add a test for block rewards with a decimal number

Reply:
yeah sure.